aiokafka/message_accumulator.py

```diff
@@ -181,12 +181,18 @@ def set_api_version(self, api_version):
         self._api_version = api_version
 
     @asyncio.coroutine
-    def close(self):
-        self._closed = True
+    def flush(self):
         # NOTE: we copy to avoid mutation during `yield from` below
+        self._closed = True
```

Comment:
Why do you set `self._closed` flag here? It's not needed I think.

Reply:
The idea was to avoid buffering more messages while flushing but now that I think of it that might not be needed.